[DNS] Navigation and overview audit (#16946)
cloudflare/cloudflare-docs: src/content/docs/dns/index.mdx

## Features

<Feature header="DNS records" href="/dns/manage-dns-records/how-to/create-dns-records/">

> Managing DNS records allows you to make resources in your domain reachable, and to configure services such as email.

Reviewer: ```suggestion
DNS records make resources served by your domain reachable, and allow you to configure services such as email.
```
